digdag-core/src/main/java/io/digdag/core/database/AESGCMSecretCrypto.java

```diff
@@ -55,18 +55,19 @@
     private static final int GCM_TAG_LENGTH = 16;
 
     private static final int TERM = 1;
-    private static final int VERSION = 1;
+    private static final int VERSION_1 = 1;
+    private static final int VERSION_2 = 2;
```

Comment:
[minor] Comparing the old and new formats, the format itself isn't changed although the limitation of field length is changed. So they are compatible and we don't need to create a new version?